deploy/testing-integration.yml

```
@@ -145,7 +145,7 @@ parameters:
 - name: IQE_TEST_IMPORTANCE
   value: ''
 - name: IQE_SEL_IMAGE
-  value: 'quay.io/redhatqe/selenium-standalone:ff_91.9.1esr_chrome_103.0.5060.114'
+  value: 'quay.io/redhatqe/selenium-standalone:ff_102.9.0esr_chrome_112.0.5615.121'
```

I would probably try the latest as the one to go instead of these specific names.

```suggestion
  value: 'quay.io/redhatqe/selenium-standalone:latest'
```